projects/ngx-charts-on-fhir/src/lib/fhir-data/fhir-data.service.spec.ts

```diff
@@ -113,6 +118,16 @@ describe('FhirDataService', () => {
     }));
   });
 
+  it('shold create a fhir resource for add Blood Pressure', async () => {
+    const resource = service.createResourceData(bloodPressure);
+    if (resource.component) {
+      const diastolicBP = resource.component[0].valueQuantity?.value;
+      const systolicBP = resource.component[1].valueQuantity?.value;
+      expect(diastolicBP).toEqual(bloodPressure.diastolic)
+      expect(systolicBP).toEqual(bloodPressure.systolic)
+    }
+  })
```

Comment:
Be careful when putting `expect()` inside an `if`. This test will pass if `createResourceData` returns null.